e2e/explorer/api/src/app/etherscan/account.ts

```diff
@@ -170,3 +170,38 @@ export function tokenNftTx(params: GetParams<TokenNftTx>) {
   if (!params.offset || !page) return etherscanTxs;
   return etherscanTxs.slice(offset*(page-1), offset*page);
 }
+
+export function token1155Tx(params: GetParams<Token1155Tx>) {
+  const {address, contractaddress, startblock = 0, endblock, page, sort='asc', offset} = params;
+  if (!address || !contractaddress) throw new Error('Error! Missing address or contract address');
+
+  const transferId = id('TransferSingle(address, address, address, uint256, uint256)');
+  const transferIdBatch = id('TransferBatch(address, address, address, uint256, uint256)')
```

Comment:
Rename + not the right interface
```suggestion
  const transferSingleId = id('TransferSingle(address, address, address, uint256, uint256)');
  const transferBatchId = id('TransferBatch(address, address, address, uint256[], uint256[])')
```